docs/guides/component-testing/angular/examples.mdx

```diff
@@ -193,6 +193,192 @@ in the future
+
+```typescript
+let titleProp = 'Test Component'
+cy.mount(TestComponent, {
+  componentProperties: {
+    title: titleProp,
+  },
+})
+```
+
+Under the hood in the `@cypress/angular-signals` test harness is really:
+
+```typescript
+cy.mount(TestComponent, {
+  componentProperties: {
+    title: signal('Test Component'),
+  },
+})
+```
+
+##### Writable Signal
+
+Since the infered generic type in our example above lives outside the Angular context, there is no way for Cypress to enforce the one-way databinding on context updates. In other words, updates to the `titleProp` variable will not update the prop being passed into `TestComponent`.
+
+To solve this problem, Cypress allows users to pass a writable `signal()` wrapped in the generic type as the prop. This allows us to test our one-way databinding.
+
+```typescript
+const myTitlePropAsSignal = signal('Test Component')
+cy.mount(TestComponent, {
+  componentProperties: {
+    title: myTitlePropAsSignal,
+  },
+})
+
+cy.get('the-title-element').should('have.text', 'Test Component')
+cy.then(() => {
+  // now set the input() through a signal to update the one-way binding
+  myTitlePropAsSignal.set('FooBar')
+})
+
+cy.get('the-title-element').should('have.text', 'FooBar')
+```
+
+###### Change spy for signal `input()`
+
+Since the prop in question is an `input()`, Cypress does not propagate changes to the `titleChange` spy.
+
+```typescript
+cy.mount(TestComponent, {
+  componentProperties: {
+    title: 'Test Component',
+    // @ts-expect-error
+    titleChange: createOutputSpy('titleChange'),
+  },
+})
+
+// some action that changes the title prop...
+
+// this assertion will not be true. Cypress will never emit change events for props that are `input()` signals
+cy.get('@titleChange').should('have.been.called')
+```
+
+#### Testing Model Signals
+
+`model()` signals are writable signals, meaning they have the ability to support two-way data binding.
+
+##### Infered Generic Type
+
+In the example below, the `count` prop being passed into our `TestComponent` is the generic type of the `model()` signal. In this example, that generic type is a `number`. Cypress sets the generic value in the writable `model()`.
+
+```ts
+cy.mount(TestComponent, {
+  componentProperties: {
+    count: 1,
+  },
+})
+
+cy.get('the-count-element').should('have.text', '1')
+```
+
+##### Writable Signal
+
+Since the infered generic type in our example above lives outside the Angular context, there is no way for Cypress to enforce the databinding on context updates. In other words, updates to the `count` variable will not update the prop being passed into `TestComponent`. This is incredibly similar to how infered generic types of `input()` signals work.
```

Comment:
```suggestion
Since the inferred generic type in our example above lives outside of the Angular context, there's no way for Cypress to enforce the data binding on context updates. In other words, updates to the `count` variable will not update the prop being passed into `TestComponent`. This is incredibly similar to how inferred generic types of `input()` signals work.
```

Reply:
addressed in 1055c7068